plugins/che-plugin-analytics/che-plugin-analytics-wsmaster/src/main/java/com/redhat/che/plugin/analytics/wsmaster/AnalyticsWsMasterModule.java

```diff
@@ -53,5 +54,18 @@ public boolean matches(Method m) {
           }
         },
         startNumberSetterInterceptor);
+
+    final DisableAuthenticationInterceptor disableAuthenticationInterceptor =
+        new DisableAuthenticationInterceptor();
+    requestInjection(disableAuthenticationInterceptor);
+    bindInterceptor(
+        Matchers.subclassesOf(AbstractKeycloakFilter.class),
+        new AbstractMatcher<Method>() {
```

Comment:
lambda usage ?

Reply:
`AbstractMatcher` is a class, not an interface, and the underlying interface has several methods.